auth/identity_context.go

```diff
@@ -81,6 +86,15 @@ func (c IdentityContext) AuthenticatedAt() time.Time {
 	return c.authenticatedAt
 }
 
+func (c IdentityContext) UserIdentifier() string {
+	return c.userIdentifier
+}
```

Comment:
This is confusing when we have `UserID()` can you more explicitly name this?

Reply:
how about `executionUserIdentifier`